src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -87,6 +87,7 @@ public ResponseEntity<String> putUserCommonsByIdBuy(
         if(userCommons.getTotalWealth() >= commons.getCowPrice() ){
           userCommons.setTotalWealth(userCommons.getTotalWealth() - commons.getCowPrice());
           userCommons.setNumOfCows(userCommons.getNumOfCows() + 1);
+          userCommons.setTotalCowHealth(userCommons.getTotalCowHealth() + 100);
```

Comment:
We could consider keeping track of averageCowHealth instead of totalCowHealth in the userCommons, since it seems that that metric is used more often. (from https://github.com/ucsb-cs156-w22/HappierCows/blob/main/docs/gamePlay.md). Although as is it is right now, it works just fine.

Reply:
I considered doing average cow health too, but I just wanted to get it working in this PR without using "complicated" math. I probably have to change it anyways when I have to fix how the wealth calculation works when selling anyways. Right now the selling price is always constant with `commons.getCowPrice()` instead of scaling with the current cow health.

Reply:
Will probably change it to average cow health when I fix issue #47, which is dealing with the backend math/calculations.

Reply:
Sounds good!